### PR TITLE
Updated landing page for week 6 prototype

### PR DESCRIPTION
**Pull request by Jay-kew**
Updated the information on `index.html` according to the landing page sections mentioned in Week 6

- a description of your user group and/or your problem statement
- which projects each person worked on
- any additional instructions or notes to users

On another note, we should come up with a good website name soon :)

### CODE REVIEW

**Review by LISH0038 — APPROVED** [review]
Thanks a lot Jingqi!! Love these words! 
My suggestion of website name: Food Blogger's Home??😂

File: src/main/webapp/index.html [hunk 1]
```diff
@@ -84,13 +85,11 @@ <h2 class="section-heading mb-4">
...
-              their URL.</p>
-              <p class="mb-0">This is your code now! Feel free to make changes, and don't be afraid to get creative!
-              You could start by modifying this page to tell the world more about your team.</p>
+              <p>Welcome to [website name]! Sign up now to join our community and get the latest updates on everything food related!</p>
+              <p>You can search through our heat map to find the most popular stalls to try. You can also post your food journey and delectable experiences with pictures and locations to our friendly community and help other food enthusiasts locate their next meal. Feel free to navigate through this website to discover more!</p>
+              <p>This awesome website was made possible by CodeU Team 45 consisting of Jingqi, Caleb, Shanlan, Yuxin and our wonderful advisor, Minqi. The user profile and image capabilities were done by Jingqi, while the stats page and styling were the works of Caleb. Shanlan and Yuxin worked on the public feed and community page respectively, and then combined their knowledge for the location and mapping functions.</p>
```
Comment (calebmah) [inline src/main/webapp/index.html:92]: Perhaps we can edit our about us page to include info such as this, the existing about us page is a little dated. Then also link to the page from here?

File: src/main/webapp/index.html [hunk 2]
```diff
@@ -84,13 +85,11 @@ <h2 class="section-heading mb-4">
...
                 <span class="section-heading-upper">CodeU</span>
-                <span class="section-heading-lower"> Starter Project</span>
+                <span class="section-heading-lower">[Website Name]</span>
               </h2>
-              <p>This is the CodeU starter project. Click the links above to login and visit your page.
-              You can post messages on your page, and you can visit other user pages if you have
-              their URL.</p>
-              <p class="mb-0">This is your code now! Feel free to make changes, and don't be afraid to get creative!
-              You could start by modifying this page to tell the world more about your team.</p>
+              <p>Welcome to [website name]! Sign up now to join our community and get the latest updates on everything food related!</p>
```
Comment (calebmah) [inline src/main/webapp/index.html:90]: Maybe we can include a link to sign up/login?

File: src/main/webapp/index.html [hunk 3]
```diff
@@ -84,13 +85,11 @@ <h2 class="section-heading mb-4">
...
+              <p>You can search through our heat map to find the most popular stalls to try. You can also post your food journey and delectable experiences with pictures and locations to our friendly community and help other food enthusiasts locate their next meal. Feel free to navigate through this website to discover more!</p>
```
Comment (calebmah) [inline src/main/webapp/index.html:91]: And also links to all these features?

**Review by calebmah — COMMENTED** [review]
Thanks for writing these! Some name ideas from the top of my head in no particular order:
- Food Haven
- Hawker Haven
- SG Food
- Instafood
- Foodiegram
- Fantastic Feasts (and where to find them)
- Makan Map
- Makan Marker
- Foodspot

**Comment by LISH0038** [comment]
> Thanks for writing these! Some name ideas from the top of my head in no particular order:
> 
> * Food Haven
> * Hawker Haven
> * SG Food
> * Instafood
> * Foodiegram
> * Fantastic Feasts (and where to find them)
> * Makan Map
> * Makan Marker
> * Foodspot

Love the idea "Fantastic Feasts (and where to find them)" 😄

**Review by BenMQ — APPROVED** [review]

File: src/main/webapp/index.html (hunk 3 above)
Comment (BenMQ) [reply src/main/webapp/index.html:91]: +1 Great idea for linking (feel free to send PR separately, I think code u team will starting looking through the sites so getting this merged earlier will help them understanding what we have)

File: src/main/webapp/index.html (hunk 2 above)
Comment (Jay-kew) [reply src/main/webapp/index.html:90]: Yup okay! Technically they are logging in with their gmail right, do we need to direct them to the gmail website to sign up for a google account if they do not have?

File: src/main/webapp/index.html (hunk 3 above)
Comment (Jay-kew) [reply src/main/webapp/index.html:91]: Ahh ok sure, I will merge this PR first then and open a new PR later with the changes :)

**Pull request merged**